src/appMain/sdl_preloaded_pt.json

```diff
@@ -1,6 +1,8 @@
 {
     "policy_table": {
         "module_config": {
+            "country_consent_passengersRC": true,

```

Comment:
@yang1070 I agree, we should remove all driver/passenger staff. It requires some code and test data changes